common/src/program_storage.rs

```diff
@@ -41,8 +41,9 @@ pub trait Error {
 pub trait ProgramStorage {
     type InternalError: Error;
     type Error: From<Self::InternalError> + Debug;
+    type BlockNumber;
```

Comment:
it would be nitce  to bind the type with some traits (Add, Zero, etc)

Reply:
If we don't use such functionality atm than it's useless for now